src/packages/table/table.scss

```
@@ -135,7 +135,7 @@
 
     &-body {
       /* #ifndef dynamic*/
```

Stylelint (17.11.1)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)